public/plugin_helpers/plugin_overview.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { ContentManagementPluginSetup } from '../../../../src/plugins/content_management/public';
+
+export const HOME_PAGE_ID = 'observabilityOverviewPage';
+export enum SECTIONS {
+  GET_STARTED = 'get_started',
+  SELECTOR = 'selector',
+  DASHBOARD = 'dashboard',
+}
+
+export enum HOME_CONTENT_AREAS {
+  GET_STARTED = `${HOME_PAGE_ID}/${SECTIONS.GET_STARTED}`,
+  SELECTOR = `${HOME_PAGE_ID}/${SECTIONS.SELECTOR}`,
+  DASHBOARD = `${HOME_PAGE_ID}/${SECTIONS.DASHBOARD}`,
+}
+
+export const setupOverviewPage = (contentManagement: ContentManagementPluginSetup) => {
+  contentManagement.registerPage({
+    id: HOME_PAGE_ID,
+    title: 'Home',
+    sections: [
+      {
+        id: SECTIONS.SELECTOR,
+        order: 2000,
+        title: 'Selector Section',
+        kind: 'custom',
+        render: (contents) => (
+          <>
+            {contents.map((content, index) =>
+              content.kind === 'custom' ? (
```

Comment:
For my knowledge what is the `custom` kind here?

Reply:
The custom part is for registering content that differs from the pre-defined cards or dashboards. In this case we are using the custom part to make the search bar, modal pop up for selection, and date selector. The framework did not exist for this functionality so we use the custom type. 
<img width="1118" alt="Screenshot 2024-08-29 at 9 37 32 AM" src="https://github.com/user-attachments/assets/9d0c8c32-c370-4955-b516-513708aaf832">